pkg/meta/version_service.go

```diff
@@ -55,7 +70,7 @@ type VersionService interface {
 	UpdateIfGreater(ctx context.Context, obj dom.Object, storage string, version int64) error
 	// DeleteObjAll deletes all obj meta for all storages
 	DeleteObjAll(ctx context.Context, obj dom.Object) error
-	// DeleteBucketMeta deletes meta of bucket objects for given storage.
+	// DeleteBucketMeta deletes all versions info for bucket and all bucket objects
```

Comment:
Is it really deletes everything in bucket or just metadata?

Reply:
thanks, it was really poor wording. there are bucket version metadata in redis and obj version metadata in redis. here we remove only metadata in redis. fixed.